File: python/15.py
def create_character()
    name = input("Enter your character's name: ")
    character_class = input("Choose a class (Warrior, Mage, Rogue): ")
    level = int(input("Enter starting level: "))
    health = 100 * level
    print("\nCharacter created:")
    print("Name:" name)
    print("Class:" character_class)
    print("Level:" level)
    print("Health:" health)

#Sample Output

"""
Enter your character's name: A
Choose a class (Warrior, Mage, Rogue): MAGE
Enter starting level: 3

Character created:
Name: A
Class: MAGE
Level: 3
Health: 300
"""

File: java/15.java
import java.util.Scanner;

public class CharacterCreator {
    public static void main(String[] args) {
        createCharacter();
    }

    public static void createCharacter() {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter your character's name: ");
        String name = scanner.nextLine();

        System.out.print("Choose a class (Warrior, Mage, Rogue): ");
        String characterClass = scanner.nextLine().toUpperCase();

        System.out.print("Enter starting level: ");
        int level = scanner.nextInt();
        int health = 100 / level; 

        System.out.println("\nCharacter created:");
        System.out.println("Name: " + name);
        System.out.println("Class: " + characterClass);
        System.out.println("Level: " + level);
        System.out.println("Health: " + health);

        scanner.close(); 
    }
}

    }
}
//sample output
/*
Enter your character's name: aravind
Choose a class (Warrior, Mage, Rogue): mage
Enter starting level: 500

Character created:
Name: aravind
Class: MAGE
Level: 500
Health: 50000
 */